qiskit/circuit/library/standard_gates/ms.py

```diff
@@ -12,22 +12,22 @@
 
 """Global Mølmer–Sørensen gate."""
 
+import warnings
 from qiskit.circuit.gate import Gate
 from qiskit.circuit.quantumregister import QuantumRegister
 
 
 class MSGate(Gate):
-    """Global Mølmer–Sørensen gate.
-
-    The Mølmer–Sørensen gate is native to ion-trap systems. The global MS can be
-    applied to multiple ions to entangle multiple qubits simultaneously.
-
-    In the two-qubit case, this is equivalent to an XX(theta) interaction,
-    and is thus reduced to the RXXGate.
+    """MSGate has been deprecated.
+    Please use ``GMS`` in ``qiskit.circuit.generalized_gates`` instead.
     """
 
     def __init__(self, num_qubits, theta, label=None):
         """Create new MS gate."""
+        warnings.warn('The qiskit.circuit.library.standard_gates.ms import '
+                      'is deprecated as of 0.16.0. You should import MSGate '
+                      'using qiskit.circuit.library.standard_gates '
```

Comment:
```suggestion
                      'using qiskit.circuit.library.generalized_gates '
```

Reply:
I'll get on these today.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/4979/commits/29bbc333d01f453f93a4970271eec7091e2fe404